src/metric-engine/src/engine/create.rs

```diff
@@ -529,7 +542,8 @@ impl MetricEngineInner {
                 DATA_SCHEMA_TSID_COLUMN_NAME,
                 ConcreteDataType::uint64_datatype(),
                 false,
-            ),
+            )
+            .with_inverted_index(false),
```

Comment:
Let's clean up https://github.com/GreptimeTeam/greptimedb/blob/7eaabb3ca232be4135c3d31df567751e743c8d3f/src/metric-engine/src/engine/options.rs#L36-L39

Reply:
Resolved in [fbed01e](https://github.com/GreptimeTeam/greptimedb/pull/5374/commits/fbed01e22f13943a6684b0f7b6f45428e4f77312)